webpack.config.js

```diff
@@ -25,9 +25,9 @@ const EXPLORER_APP = {
     EXPLORER_APP_DOMAIN_REGEX_PROD: '^explorer\\.cow\\.fi|^gnosis-protocol\\.io',
     EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.explorer\\.cow\\.fi|^barn\\.gnosis-protocol\\.io',
 
-    OPERATOR_URL_STAGING_MAINNET: 'https://protocol-mainnet.dev.gnosisdev.com/api',
-    OPERATOR_URL_STAGING_RINKEBY: 'https://protocol-rinkeby.dev.gnosisdev.com/api',
-    OPERATOR_URL_STAGING_XDAI: 'https://protocol-xdai.dev.gnosisdev.com/api',
+    OPERATOR_URL_STAGING_MAINNET: 'https://barn.api.cow.fi/mainnet/api',
```

Comment:
@alfetopito I would like to clarify, will be `/apli` also be required here?

Reply:
Yes, exactly like api.cow.fi